CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 0.8.9
```

Comment:
This should be `0.9.1-dev0` now, and you'll need to do `unstructured/__version__.py` as well.